frontend/src/metabase/admin/settings/components/widgets/PublicLinksListing/PublicLinksListing.jsx

```diff
@@ -142,6 +151,25 @@ export const PublicLinksQuestionListing = () => (
   />
 );
 
+const mapStateToProps = state => ({
+  siteUrl: getSetting(state, "site-url"),
+});
+
+export const PublicLinksActionListing = connect(mapStateToProps)(
+  ({ siteUrl }) => (
```

Comment:
I think this way `displayName` wouldn't be set for a component. Could you move this to a separate component?

Reply:
Ahh, haven't thought about that. Good call.